fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/ShowCreateViewCommand.java

```diff
@@ -0,0 +1,127 @@
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.nereids.trees.plans.commands;
+
+import org.apache.doris.analysis.StmtType;
+import org.apache.doris.catalog.Column;
+import org.apache.doris.catalog.DatabaseIf;
+import org.apache.doris.catalog.Env;
+import org.apache.doris.catalog.ScalarType;
+import org.apache.doris.catalog.TableIf;
+import org.apache.doris.catalog.View;
+import org.apache.doris.common.ErrorCode;
+import org.apache.doris.common.ErrorReport;
+import org.apache.doris.nereids.trees.plans.PlanType;
+import org.apache.doris.nereids.trees.plans.commands.info.TableNameInfo;
+import org.apache.doris.nereids.trees.plans.visitor.PlanVisitor;
+import org.apache.doris.qe.ConnectContext;
+import org.apache.doris.qe.ShowResultSet;
+import org.apache.doris.qe.ShowResultSetMetaData;
+import org.apache.doris.qe.StmtExecutor;
+
+import com.google.common.collect.Lists;
+
+import java.util.List;
+
+/**
+ * Represents the command for SHOW CREATE VIEW.
+ */
+public class ShowCreateViewCommand extends Command implements NoForward {
+    private static final ShowResultSetMetaData VIEW_META_DATA =
+            ShowResultSetMetaData.builder()
+                    .addColumn(new Column("View", ScalarType.createVarchar(20)))
+                    .addColumn(new Column("Create View", ScalarType.createVarchar(30)))
+                    .addColumn(new Column("character_set_client", ScalarType.createVarchar(30)))
+                    .addColumn(new Column("collation_connection", ScalarType.createVarchar(30)))
+                    .build();
+
+    private final String catalog;
+    private final String dbName;
+    private final String viewName;
+
+    public ShowCreateViewCommand(TableNameInfo tableNameInfo) {
+        super(PlanType.SHOW_CREATE_VIEW_COMMAND);
+        this.catalog = tableNameInfo.getCtl();
+        this.dbName = tableNameInfo.getDb();
+        this.viewName = tableNameInfo.getTbl();
+    }
+
+    public String getCatalog() {
+        return catalog;
+    }
+
+    public String getDbName() {
+        return dbName;
+    }
+
+    public String getViewName() {
+        return viewName;
+    }
+
+    @Override
+    public void run(ConnectContext ctx, StmtExecutor executor) throws Exception {
```

Comment:
ShowCreateTableStmt's analyze method needs to be re-implemented in nereids. For example, add a private void validate(ConnectContext ctx) throws UserException method to do the job, and call it at the beginning of run method. BTW, the TableNameInfo class has analyze method to do the validation. You'd better keep TableNameInfo as a class member and call its analyze method in the new validate function

Reply:
Thanks for your comment. I have updated the PR, please review